bundles/org.openhab.binding.cul.max/README.md

```diff
@@ -0,0 +1,178 @@
+| cul_max_bridge | Bridge | CUL if using a serial CUL
+| cun_max_bridge | Bridge | CUN if using a network CUL
+| maxcul_bridge  | Bridge | This is the MAX! Cube emulation for the CUL.                                                                                 |
+| thermostat     | Thing  | This is for the MAX! Heating Thermostat. This is also used for the power plug switch "Zwischenstecker-Schaltaktor". |
+| thermostatplus | Thing  | This is for the MAX! Heating Thermostat+. This is the type that can hold the program by itself.                    |
+| wallthermostat | Thing  | MAX! Wall Thermostat.                                                                                              |
+| ecoswitch      | Thing  | MAX! Ecoswitch.                                                                                                    |
+| shuttercontact | Thing  | MAX! Shuttercontact / Window Contact.                                                                              |
+
+Generally one does not have to worry about the thing types as they are automatically discovered.
+If for any reason you need to manually define the Things and you are not exactly sure what type of thermostat you have, you can choose `thermostat` for both the thermostat and thermostat+, this will not affect their working.
+
+## Discovery
+
+Depending on your you have to add a `cul_max_bridge` for serial CUL or a `cun_max_bridge` for a network CUN first as a bridge
+for the `maxcul_bridge` which you have to add next to be ready for discovery of you MAX! devices.
+
+Start [Pairing](#Pairing) at your MAX! device. It will show up in the inbox and you can add it to your configuration. Serial number and rfAddress is set already.
+After adding it to your configuration you have to set channel `pair_mode` to ON on your `maxcul_bridge` and restart
+the pairing on your MAX! device to start the pairing sequence. 
+If the device is already paired and starts re-pairing e.g. after battery change, the state of channel pair_mode is ignored.
+
+## Binding Configuration
+
+The binding requires access to the serial device connecting to the CUL device as described in https://www.openhab.org/docs/administration/serial.html
+when using a `maxcul_bridge`.
+
+## Thing Configuration
+
+### CUL Bridge Configuration
+
+| Property | Default | Required | Description |
+|----------|---------|:--------:|-------------|
+| device   |         |   Yes    | in the form `<device>`, where `<device>` is a local serial port eg `/dev/ttyACM0` |
+| baudrate |         |   No     | one of 75, 110, 300, 1200, 2400, 4800, 9600, 19200, **38400**, 57600, 115200 |
+| parity   |         |   No     | one of EVEN, ODD, MARK, **NONE**, SPACE |
+
+When using a serial port, you may need to add `-Dgnu.io.rxtx.SerialPorts=/dev/ttyACM0` in your server startup.  Please consult the [forum](https://community.openHAB.org) for the latest information.

```

Comment:
You can replace this sentence by a reference to the generic serial port configuration, which gives some more detailed information: https://www.openhab.org/docs/administration/serial.html#serial-port-configuration